python/nano/example/pytorch/inference_pipeline/vision_transformers/vision_transformer.ipynb

```diff
@@ -264,21 +266,21 @@
       "|             method             |        status        | latency(ms)  |       accuracy       |\n",
       " -------------------------------- ---------------------- -------------- ----------------------\n",
-      "|            original            |      successful      |    24.861    |        0.666         |\n",
-      "|           fp32_ipex            |      successful      |    27.165    |    not recomputed    |\n",
+      "|            original            |      successful      |    24.897    |        0.684         |\n",
+      "|           fp32_ipex            |      successful      |    26.751    |    not recomputed    |\n",
       "|              bf16              |   fail to forward    |     None     |         None         |\n",
-      "|           bf16_ipex            |      successful      |   174.285    |        0.666         |\n",
-      "|              int8              |      successful      |    25.602    |        0.006         |\n",
-      "|            jit_fp32            |      successful      |    20.241    |    not recomputed    |\n",
-      "|         jit_fp32_ipex          |      successful      |    19.565    |    not recomputed    |\n",
-      "|  jit_fp32_ipex_channels_last   |      successful      |    15.542    |    not recomputed    |\n",
-      "|         openvino_fp32          |      successful      |    10.614    |    not recomputed    |\n",
-      "|         openvino_int8          |      successful      |    80.305    |        0.662         |\n",
-      "|        onnxruntime_fp32        |      successful      |    12.602    |    not recomputed    |\n",
-      "|    onnxruntime_int8_qlinear    |      successful      |    11.743    |        0.006         |\n",
+      "|           bf16_ipex            |      successful      |   178.762    |        0.684         |\n",
+      "|              int8              |      successful      |    25.912    |        0.003         |\n",
+      "|            jit_fp32            |      successful      |    19.275    |    not recomputed    |\n",
+      "|         jit_fp32_ipex          |      successful      |    18.964    |    not recomputed    |\n",
+      "|  jit_fp32_ipex_channels_last   |      successful      |    15.155    |    not recomputed    |\n",
+      "|         openvino_fp32          |      successful      |    10.464    |    not recomputed    |\n",
+      "|         openvino_int8          |      successful      |    83.362    |        0.653         |\n",
+      "|        onnxruntime_fp32        |      successful      |    12.774    |    not recomputed    |\n",
+      "|    onnxruntime_int8_qlinear    |      successful      |    11.523    |        0.003         |\n",
       "|    onnxruntime_int8_integer    |   fail to convert    |     None     |         None         |\n",
       " -------------------------------- ---------------------- -------------- ----------------------\n",
-      "\n"
+      "Optimization cost 1.82e+02s at all.\n"
```

Comment:
Why scientific notation?

Change to `Optimization cost 182s in total`

Reply:
done